.github/workflows/centraldb_manifests_build.yaml

```diff
@@ -0,0 +1,23 @@
+name: Build CentralDashboard manifests
+on:
+  pull_request:
+    paths:
+      - components/centraldashboard/**
```

Comment:
Since we are testing manifests this time we don't want to trigger the workflows when code changes, but rather when the **manifests** change.

So in this case it will be `components/centraldashboard/manifests/**`

Reply:
I made the change.